### PR TITLE
feat: [STO-6015]: noop on CI License check for STOManager

### PR DESCRIPTION
**Pull request by ahokinson**
## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- 
  Specific builds
  - `trigger manager`
  - `trigger dms`
  - `trigger ng_manager`
  - `trigger cvng `
  - `trigger cmdlib`
  - `trigger template_svc`
  - `trigger events_fmwrk_monitor`
  - `trigger event_server`
  - `trigger change_data_capture`
  -  Trigger multiple builds together: For eg: `trigger dms, manager`
- Immutable delegate `trigger publish-delegate`
</details>

## [Latest PR Check Triggers](https://github.com/harness/harness-core/blob/develop/.github/pull_request_template.md)

<details>
  <summary>PR Check triggers</summary>
You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ut0, ut1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- Trigger CommonChecks: `trigger commonchecks`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger uts`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
- Go Build: `trigger gobuild`
- Validate_Reviews: `trigger review`
- Module Dependency Check: `trigger mdc`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/50709)

### CODE REVIEW

**Review by r0610205 — APPROVED** [review]

**Review by r0610205 — DISMISSED** [review]

**Comment by r0610205** [comment]
trigger uts

**Comment by r0610205** [comment]
trigger ut5

**Comment by ahokinson** [comment]
trigger uts

File: 315-sto-manager/app/src/main/java/io/harness/license/STOLicenseNoopServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import io.harness.licensing.beans.summary.LicensesWithSummaryDTO;
+
+@OwnedBy(STO)
+public class STOLicenseNoopServiceImpl implements CILicenseService {
+  @Override
+  public LicensesWithSummaryDTO getLicenseSummary(String accountId) {
+    return CILicenseSummaryDTO.builder()
+        .edition(Edition.ENTERPRISE)
+        .licenseType(LicenseType.PAID)
+        .maxExpiryTime(2147483647) // maximum positive value for a 32-bit signed integer
```
Comment (jamie-harness) [inline 315-sto-manager/app/src/main/java/io/harness/license/STOLicenseNoopServiceImpl.java:27]: Integer.MAX_VALUE

**Review by jamie-harness — APPROVED** [review]
Ideally, we should have a logic to check in common steps. But this is the fastest way to unblock. Approving now, but please create a followup ticket on further separate two services

**Comment by ahokinson** [comment]
trigger uts

**Review by jamie-harness — APPROVED** [review]

**Review by jamie-harness — APPROVED** [review]

**Comment by ahokinson** [comment]
trigger uts

**Review by r0610205 — APPROVED** [review]

**Comment by ahokinson** [comment]
trigger uts

**Review by jamie-harness — APPROVED** [review]

**Review by r0610205 — APPROVED** [review]

**Pull request merged**